packages/core/src/components/tree/tree.md

```diff
@@ -33,7 +33,7 @@ example, `[2, 0]` represents the first child (`0`) of the third top-level node (
 
 @### Instance methods
 
-<div class="docs-interface-name">Tree</div>
+<div class="@ns-monospace-text">Tree</div>
```

Comment:
use backticks. `Tree`. should be a `code` element.

Reply:
I don't want a code tho. I want a large text monospace element, without the border. Also `<code>` always enforces 12px so can't be combined with `.pt-text-large` :(